### PR TITLE
fix(threaded-replies): fix styling for replies

### PR DESCRIPTION
**Pull request by wpiesiak**
- Round up the gray vertical line for replies in thread
- Make the “hide replies” and “show replies” button to be bold
- Make the “Reply” button to be bolded, and be $bdl-gray-62 (#767676)
- Decrease the space between replies
- Decrease space above "See more replies" button to 16px

### CODE REVIEW

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplyForm.scss [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+.btn-plain.bcs-ActivityThreadReplyForm-toggle {
     display: flex;
     align-items: center;
+    color: $bdl-gray-62;
+    font-weight: bold;
 
     &,
     &:hover {
         padding: $bdl-grid-unit $sidebarActivityFeedSpacingHorizontal ($bdl-grid-unit * 5);
+        font-weight: bold;
```
Comment (jankowiakdawid) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplyForm.scss:12]: Can we add `focus-visible` styles here also? Currently there's no visible cue when one navigates with keyboard in sidebar that they on Reply button.

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
-.bcs-ActivityThread-toggle {
-    margin-left: $sidebarActivityFeedSpacingHorizontal;
-    color: $bdl-box-blue;
-    font-weight: bold;
-
-    &:hover,
-    &:active {
+.bcs-ActivityThread {
+    .bcs-ActivityThread-toggle {
+        margin-top: -9px;
```
Comment (karelee7) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss:5]: minor: Do we typically use $bdl-grid-unit for margin units?? I've seen decimal multipliers used in some places, but not sure if this is standard?

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplies.scss [hunk 3]
```diff
@@ -11,6 +11,7 @@
         left: $sidebarActivityFeedSpacingHorizontal;
         width: 4px;
         background: $bdl-gray-10;
+        border-radius: 2px;
```
Comment (karelee7) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplies.scss:14]: Can we use any of the `$bdl-border-radius...` variables here?

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss [hunk 4]
```diff
@@ -1,14 +1,18 @@
...
         margin-left: $sidebarActivityFeedSpacingHorizontal;
+        color: $bdl-box-blue;
         font-weight: bold;
+
+        &:hover,
+        &:active {
+            margin-top: -9px;
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss:12]: Can we add a brief comment as to why we selected this value? Is it based on another style that could change independently and cause a visual regression?

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss [hunk 5]
```diff
@@ -1,14 +1,18 @@
...
-    &:hover,
-    &:active {
+.bcs-ActivityThread {
+    .bcs-ActivityThread-toggle {
+        margin-top: -9px;
         margin-left: $sidebarActivityFeedSpacingHorizontal;
+        color: $bdl-box-blue;
         font-weight: bold;
+
+        &:hover,
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss:10]: Minor, but the duplicate in the base and hover/active states can be solved:

```
&,
&:hover,
&:active {
   // shared styles
}

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplyForm.scss [hunk 6]
```diff
@@ -1,12 +1,15 @@
 @import '../../../common/variables';
 
-.bcs-ActivityThreadReplyForm-toggle {
+.btn-plain.bcs-ActivityThreadReplyForm-toggle {
     display: flex;
     align-items: center;
+    color: $bdl-gray-62;
+    font-weight: bold;
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplyForm.scss:7]: This style is not needed, as it's covered by the `&` directive below.

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss [hunk 7]
```diff
@@ -1,14 +1,25 @@
...
+        }
+    }
+
+    .bcs-ActivityCard {
+        margin: $sidebarActivityFeedSpacingVertical $sidebarActivityFeedSpacingHorizontal;
+        padding: 0;
+    }
+
+    button:focus-visible {
+        outline: -webkit-focus-ring-color auto 1px;
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss:22]: I don't think I've seen this before. Is this a standard visual treatment? Can we reuse the same treatment as hover and active, instead?

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss [hunk 8]
```diff
@@ -1,14 +1,25 @@
...
+            font-weight: bold;
+        }
+    }
+
+    .bcs-ActivityCard {
+        margin: $sidebarActivityFeedSpacingVertical $sidebarActivityFeedSpacingHorizontal;
+        padding: 0;
+    }
+
+    button:focus-visible {
```
Comment (jstoffan) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss:21]: Can we target a CSS class?

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss (hunk 7 above)
Comment (tjuanitas) [reply src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss:22]: is an equivalent needed for Firefox?

Comment (wpiesiak) [reply src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss:22]: It's about issue Dawid mentioned: https://github.com/box/box-ui-elements/pull/3110#discussion_r1004231413

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplyForm.scss [hunk 9]
```diff
@@ -1,12 +1,14 @@
 @import '../../../common/variables';
 
-.bcs-ActivityThreadReplyForm-toggle {
+.btn-plain.bcs-ActivityThreadReplyForm-toggle {
```
[the remark this answers is not in the document]
Comment (wpiesiak) [reply src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplyForm.scss:3]: Actually the` font-weight `is being inherited from `src/elements/common/base.scss`. I've moved it to separate declaration for better visibility.

**Review by jstoffan — APPROVED** [review]

**Pull request merged**